Paintroid/src/main/java/org/catrobat/paintroid/contract/MainActivityContracts.java

```diff
@@ -244,6 +245,10 @@ void restoreState(boolean isFullscreen, boolean isSaved, boolean isOpenedFromCat
 		int getHeight();
 
 		void removeItem(int item);
```

Comment:
This can be removed from the interface, it's only used from tests and the class itself.